tests/unit/test_export_phase4.py

```
@@ -0,0 +1,938 @@
+            },
+            {
+                "pr_number": 124,
+                "success": False,
+                "result": "",  # Empty string result
+                "errors": [],  # Empty errors list
+                "duration": None  # None duration
+            }
+        ]
+
+        # Should handle None and empty values gracefully
+        summary = BatchSummary(
```

Ruff (0.13.1)

Local variable `summary` is assigned to but never used

Remove assignment to unused variable `summary`

(F841)